.gitignore

```diff
@@ -0,0 +1,27 @@
+# python related files
+*.pyc
+*.pyo
+*.egg
+*.egg-info
+__pycache__
+
+# sqlite db
+*.db
+*.sqlite3
+
+# sublime related files
+*.sublime-project
+*.sublime-workspace
+
+# test, build, log
+build
+docs/_build
+dist
+results
+logs
+*.logs
+venv
+
+# IDE
+.idea
+.vscode
```

Comment:
Add EOL (end-of-file) here - empty line at the end of the file.

Reply:
fixed